### PR TITLE
Set up release action

### PR DESCRIPTION
**Pull request by glyn**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    tags:
+      - 'v*'
+jobs:
+  build:
+    name: release action
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: build binaries
+      uses: cedrickring/golang-action/go1.12@1.3.0
```
Comment (scothis) [inline .github/workflows/main.yaml]: why use this over using the official setup-go action?

https://github.com/actions/setup-go#usage

File: Makefile [hunk 2]
```diff
@@ -24,4 +24,6 @@ irel: $(GO_SOURCES)
 release: $(GO_SOURCES) test
 	GOOS=darwin   GOARCH=amd64 go build -o $(OUTPUT)     cmd/irel/main.go && tar -czf irel-darwin-amd64.tgz  $(OUTPUT)     && rm -f $(OUTPUT)
 	GOOS=linux    GOARCH=amd64 go build -o $(OUTPUT)     cmd/irel/main.go && tar -czf irel-linux-amd64.tgz   $(OUTPUT)     && rm -f $(OUTPUT)
-	GOOS=windows  GOARCH=amd64 go build -o $(OUTPUT).exe cmd/irel/main.go && zip -mq  irel-windows-amd64.zip $(OUTPUT).exe && rm -f $(OUTPUT).exe
+	#GOOS=windows  GOARCH=amd64 go build -o $(OUTPUT).exe cmd/irel/main.go && zip -mq  irel-windows-amd64.zip $(OUTPUT).exe && rm -f $(OUTPUT).exe
```
Comment (scothis) [inline Makefile]: one advantage of not running in a container is that you can install bits into the vm and have them be available to later stages

File: .github/workflows/main.yaml (hunk 1 above)
Comment (glyn) [reply .github/workflows/main.yaml]: Thanks. I'll take a look.

File: Makefile (hunk 2 above)
Comment (glyn) [reply Makefile]: True. I'll see if that approach can be made to work.

**Comment by glyn** [comment]
@scothis So much faster and cleaner doing the make in the VM - thanks.

**Comment by glyn** [comment]
@scothis Ready for your approval now. Note: https://github.com/pivotal/image-relocation/issues/32

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  build:
+    name: release action
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.12'
+    - name: zip windows executable
```
Comment (scothis) [inline .github/workflows/main.yaml]: the name looks a bit stale given other changes

Comment (glyn) [reply .github/workflows/main.yaml]: Fixed.

**Pull request merged**